desktop/cmp/button/AppMenuButton.ts

```diff
@@ -169,16 +165,16 @@ function buildMenuItems({
     ]);
 }
 
-function parseMenuItems(items) {
-    return items
-        .filter(it => !it.omit)
+// Todo: Use MenuItemSpec type rather than Record<string,any>
```

Comment:
this is ok, as we discussed -- but given that its just a helper method, we could also just continue to pass it as any, rather than this.

The calculus might be different when we have a `MenuItemLike` type